src/main/java/nl/knaw/dans/ttv/client/mappers/OcflObjectVersionMapper.java

```diff
@@ -33,13 +33,14 @@ public interface OcflObjectVersionMapper {
     OcflObjectVersionMapper INSTANCE = Mappers.getMapper(OcflObjectVersionMapper.class);
 
     @Mapping(source = "swordClient", target = "dataSupplier")
-    @Mapping(source = "datasetPid", target = "dataversePid")
+    @Mapping(source = "doi", target = "dataversePid")
     @Mapping(source = "datasetVersion", target = "dataversePidVersion")
     // TODO confirm this is correct
     @Mapping(source = "dveFilePath", target = "ocflObjectPath")
```

Comment:
the ocflObjectPath should be the path in the ocfl object, not the path of the bag on the disk.